packages/rum-core/src/browser/performanceCollection.ts

```diff
@@ -268,11 +268,11 @@ function computeRelativePerformanceTiming() {
   const result: Partial<IndexedPerformanceTiming> = {}
   const timing = performance.timing as IndexedPerformanceTiming
   for (const key in timing) {
-    if (isNumber(timing[key])) {
-      result[key] = timing[key] === 0 ? 0 : getRelativeTime(timing[key] as number)
+    if (isNumber<TimeStamp>(timing[key])) {
+      result[key] = timing[key] === 0 ? 0 : getRelativeTime(timing[key])
```

Comment:
I would keep the explicit cast here instead of using a parametric type: `getRelativeTime(timing[key] as TimeStamp)`